froggah.py

```diff
@@ -91,7 +89,7 @@
     for e in pygame.event.get():  # Handle events: mouse, keyb etc.
         if e.type == pygame.QUIT: playing = False
     
-    for mov in range(screenx):
+    for mov in range(screenx + carwidth):

```

Comment:
again, now the screen of the cars is larger by the user defined width of the car